### PR TITLE
Delete scenario files

### LINKED ISSUE #54: Erasing debug folder (xml files, etc...) once experiment done
avoid memory overload: debug folder gets very big

### PR DESCRIPTION
**Pull request by AboudyKreidieh**
- resolves #54 
- environment automaticaly deletes files upon closing
- using timestamp instead of `time.time()` for scenario names

### CODE REVIEW

**Comment by berkeleyflow** [comment]
Have you tested that this works for rllib? Slightly worried that this will throw an error as different threads will look for the xml files but one thread has already deleted them?

File: flow/scenarios/base_scenario.py [hunk 1]
```diff
@@ -620,6 +621,30 @@ def prev_edge(self, edge, lane):
...
+        os.remove(self.generator.net_path + self.generator.nodfn)
+        os.remove(self.generator.net_path + self.generator.edgfn)
+        os.remove(self.generator.net_path + self.generator.cfgfn)
+        os.remove(self.generator.cfg_path + self.generator.addfn)
+        os.remove(self.generator.cfg_path + self.generator.guifn)
+        os.remove(self.generator.cfg_path + self.generator.netfn)
+        os.remove(self.generator.cfg_path + self.generator.roufn)
+        os.remove(self.generator.cfg_path + self.generator.sumfn)
+
+        # the connection file is not always created
```
[the remark this answers is not in the document]
Comment (AboudyKreidieh) [reply flow/scenarios/base_scenario.py]: yup, see `generate_net` for more

File: flow/core/vehicles.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 SPEED_MODES = {"aggressive": 0, "no_collide": 1, "right_of_way": 25,
                "all_checks": 31}
-LC_MODES = {"aggressive": 0, "no_lat_collide": 512, "strategic": 853}
+LC_MODES = {"aggressive": 0, "no_lat_collide": 512, "strategic": 1621}
```
[the remark this answers is not in the document]
Comment (AboudyKreidieh) [reply flow/core/vehicles.py]: this is from #61, my bad for adding it

File: tests/fast_tests/test_two_loops_one_merging.py [hunk 3]
```diff
@@ -73,7 +73,10 @@ def setUp(self):
 
     def tearDown(self):
         # terminate the traci instance
-        self.env.terminate()
+        try:
+            self.env.terminate()
+        except FileNotFoundError:
+            pass
```
[the remark this answers is not in the document]
Comment (AboudyKreidieh) [reply tests/fast_tests/test_two_loops_one_merging.py]: some tests use experiments, which will by themselves delete the scenario files (this is to not fail in those cases)

**Comment by AboudyKreidieh** [comment]
tested locally on multiple CPUs. should be good to go

**Pull request merged**